rfc/5280/test/chain-test.js

```diff
@@ -0,0 +1,193 @@
+var assert = require('assert');
+var asn1 = require('../../../');
+var rfc5280 = require('../../5280');
+var rfc5280 = require('..');
+
+var Buffer = require('buffer').Buffer;
+
+describe('asn1.js RFC5280', function() {
+  it('should decode Certificate Extensions', function() {
+    var chain = [];
+    var cert;
+    var extensions = {}
+
+    chain[0] = new Buffer(''
```

Comment:
I was thinking about it lately, what do you think about putting this blobs of data into `test/fixtures/` as just files, and reading them with `fs` here?